src/lib/scene.js

```diff
@@ -3,12 +3,18 @@ import * as THREE from 'three';
 const scene = new THREE.Scene();
 const camera = new THREE.PerspectiveCamera(75, window.innerWidth / window.innerHeight, 0.1, 1000);
 const geometry = new THREE.BoxGeometry();
-const material = new THREE.MeshBasicMaterial({ color: 0x00ff00 });
+const material = new THREE.MeshStandardMaterial({ color: 0x00ff00 });
+material.metalness = 0.1;
```

Comment:
Why wouldn't you set this in the constructor?  Also why did you chose this value opposed the example?

Reply:
I should've set it in constructor, you're right. As for the values, it's because I don't know the actual values in the example. Is there somewhere in the commit history or article that shows the values?